src/style.scss

```diff
@@ -95,17 +96,32 @@ $toolbarHeight: 50px;
     background-size: cover;
 }
 
+.caption,
 .toolbar {
     background-color: rgba(0, 0, 0, 0.5);
     position: absolute;
     left: 0;
-    top: 0;
     right: 0;
-    height: $toolbarHeight;
     display: flex;
     justify-content: space-between;
 }
 
+.caption {
+    bottom: 0;
+    max-height: $maxCaptionHeight;
+    overflow: scroll;
```

Comment:
`overflow: auto;` is preferable, as it won't display the scroll bar when the content is not scrollable.